sawmill-core/src/test/java/io/logz/sawmill/conditions/MatchRegexConditionTest.java

```diff
@@ -19,7 +21,9 @@
 
     @Before
     public void init() {
-        ConditionFactoryRegistry conditionFactoryRegistry = ConditionFactoryRegistry.getInstance();
+        ConditionFactoryRegistry conditionFactoryRegistry = new ConditionFactoryRegistry(
+                new ConditionalFactoriesLoader(new TemplateService())
+        );
```

Comment:
Use the default one from `FactoryUtils`

Reply:
I think it is better to not use the static variables if possible. 
Static variables that have state across tests can complicate things.